doc/projectBranding.md

```diff
@@ -8,6 +8,8 @@ description**, and **Cosmo Tech contact information**, but you can adapt its con
 will present how to configure minor changes to the default component, and how you can provide your own React component
 to replace the default one.
 
+[To customize about labels, see translation sections](#translations-customization)
```

Comment:
To make it clearer, I'd write _About_ with a capital _A_ or rephrase it to something like _To customize labels of this component._..

Reply:
Rephrased in the new commit :slightly_smiling_face: